cmd/executor/main_test.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/allegro/mesos-executor"
+	"github.com/mesos/mesos-go/api/v1/lib/executor/config"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestExecutorStartsWithoutCheckpointConfig(t *testing.T) {
+	// given
+	conf := executor.Config{
+		Debug: true,
+		MesosConfig: config.Config{
+			FrameworkID: "framework",
+			ExecutorID:  "executor",
+			AgentEndpoint: "localhost",
+		},
+	}
+
+	// when
+	err := startExecutor(conf)
+
+	// then
+	assert.Error(t, err)
```

Comment:
I think we need also a [NotPanics](https://godoc.org/github.com/stretchr/testify/assert#NotPanics) assertion here - we fixing bug that code is panicking when some config options are missing.